pom.xml

```diff
@@ -509,6 +509,12 @@
       </dependency>
 
       <dependency>
+        <groupId>com.google.code.findbugs</groupId>
+        <artifactId>jsr305</artifactId>
+        <version>1.3.9</version>
+      </dependency>
```

Comment:
just came to my mind: alternatively, we might place this dependency directly under `<project> <build> <dependencies> ...` this will then make every child project dependent on it. this would save some typing, but is generally considered bad practice.